docs/api/sdk/README.md

```diff
@@ -400,35 +401,45 @@ Use the `exception` type for errors and crashes.
+| -------------------------- | ------- | -------- | ------------------------------ |
+| `type`                     | string  | Yes      | Type of the exception          |
+| `message`                  | string  | Yes      | Error message text             |
+| `frames`                   | array   | Yes      | Array of stackframe objects    |
+| `signal`                   | string  | Yes      | BSD termination signal         |
+| `thread_name`              | string  | Yes      | Name of thread                 |
+| `thread_sequence`          | number  | Yes      | Thread number                  |
+| `os_build_number`          | string  | Yes      | Operating system build number  |
 
 `thread` objects
 
 Each thread object contains further fields.
+Note: Only non-crashed threads are to be send in thread object.
 
-| Field    | Type   | Optional | Comment                     |
-| -------- | ------ | -------- | --------------------------- |
-| `name`   | string | Yes      | Name of thread              |
-| `frames` | array  | Yes      | Array of stackframe objects |
+| Field      | Type   | Optional | Comment                     |
+| ---------- | ------ | -------- | --------------------------- |
+| `name`     | string | Yes      | Name of thread              |
+| `frames`   | array  | Yes      | Array of stackframe objects |
+| `sequence` | number | Yes      | Thread number               |
 
 `frame` objects
 
 Each frame object contains further fields.
 
-| Field         | Type   | Optional | Comment                        |
-| ------------- | ------ | -------- | ------------------------------ |
-| `line_num`    | number | Yes      | Line number of the method      |
-| `col_num`     | number | Yes      | Column number of the method    |
-| `module_name` | string | Yes      | Name of the originating module |
-| `file_name`   | string | Yes      | Name of the originating file   |
-| `class_name`  | string | Yes      | Name of the originating class  |
-| `method_name` | string | Yes      | Name of the originating method |
+| Field               | Type   | Optional | Comment                        |
+| ------------------- | ------ | -------- | ------------------------------ |
+| `line_num`          | number | Yes      | Line number of the method      |
+| `col_num`           | number | Yes      | Column number of the method    |
+| `module_name`       | string | Yes      | Name of the originating module |
+| `file_name`         | string | Yes      | Name of the originating file   |
+| `class_name`        | string | Yes      | Name of the originating class  |
+| `method_name`       | string | Yes      | Name of the originating method |
+| `binary_name`       | string | Yes      | Name of the iOS binary image   |
+| `symbol_address`    | string | Yes      | Address to symbolicate         |
+| `offset`            | number | Yes      | Byte offset                    |
+| `binary_address`    | string | Yes      | Binary load address            |
```

Comment:
add an `in_app` field that mentions weather the crashed stack frame binary is app binary or system binary